app/Repositories/Eloquents/FollowEloquentRepository.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/FollowEloquentRepository.php</br>**Line**: 3</br>There must be one blank line after the namespace declaration